### DISCUSSION TITLE
variable SQLEXPORTER_TARGET_DSN

### DISCUSSION THREAD

**Discussion by atchaikovski** [burningalchemist/sql_exporter#444] [Q&A] [answered]
Hello gents!
we're trying to use the image in our scripts to get metrics and seems the only option to get DB connection working is assign a proper string to the variable above. is that correct? shall we always use this var or there's another way to get connection? thank you!

**Comment by burningalchemist** [comment, marked as answer]
Hey @atchaikovski, the connection string can be configured in `sql_exporter.yml` file, or via the mentioned environment variable. 👍

If used as a container image, users are expected to mount the configuration file next to the binary or use command line flags.

Here's some info on the available env variables: https://github.com/burningalchemist/sql_exporter/pull/430

Hope it helps, let me know if you have any issues. 👍

**Comment by atchaikovski** [comment]
Thank you Sergei.
Will go the way of env vars.
Kind regards,
Alex

**Reply by burningalchemist** [reply]
Let me know if anything. 🙂 I think you still need to provide queries via the collector files. I'd suggest having a look at the example (in the `examples` folder). 👍
